frontend/components/containers/comments/CommentList.vue

```diff
@@ -122,18 +119,20 @@ export default {
     }
   },
   methods: {
-    ...mapActions('documents', ['getCommentListComplete', 'getDocumentList']),
+    ...mapActions('documents', ['getProjectCommentList']),
     ...mapMutations('documents', ['updateSelectedComments']),
 
     toLabeling(comment) {
-      const index = this.items.findIndex(item => item.id === comment.document)
-      const offset = (this.options.page - 1) * this.options.itemsPerPage
-      const page = offset + index + 1
+      const docs = []
+      for (const c in this.comments) {
+        !docs.includes(this.comments[c].document) && docs.push(this.comments[c].document)
+      }
+      const page = docs.sort().findIndex(item => item === comment.document) + 1
```

Comment:
this creates a list of all documents with comments and uses the list index of the doc corresponding to the comment as the page number. i think it can only work like this if all documents in a project have comments.

is there no way to directly link to a document page by the document id? the main problem you have seems to be that this isn't the case and the document view uses page numbers instead of ids.

Reply:
if there really is no way, then remove this feature and mention the problem in an upstream pull request later.